dm/syncer/compactor_test.go

```diff
@@ -90,6 +91,8 @@ func (s *testSyncerSuite) TestCompactJob(c *C) {
 			Length: types.UnspecifiedLength,
 		}},
 	}
+	downTi := schema.GetDownStreamTi(ti, ti)
```

Comment:
Can we use a different tableinfo in test?

Reply:
How about add some integration test in shardddl1 for compact and multiple-rows by change the origin test or add new test cases.

Reply:
i think it needs add integration test for ompact and multiple-rows too, i will do it later.